util/reflect.go

```diff
@@ -637,8 +639,9 @@ func ForEachField(schema *yang.Entry, value interface{}, in, out interface{}, it
 // may be any Go type) and executes iterFunction on each field that is present
 // within the supplied schema. Fields that are explicitly noted not to have
 // a schema (e.g., annotation fields) are skipped.
-//   in, out are passed through from the caller to the iteration and can be used
-//     arbitrarily in the iteration function to carry state and results.
+//
+//	in, out are passed through from the caller to the iteration and can be used
+//	  arbitrarily in the iteration function to carry state and results.
```

Comment:
Same here, Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.